tests/suites/test_suite_pk.function

```diff
@@ -1055,6 +1082,17 @@ void pk_sign_verify(int type, int curve_or_keybits, int sign_ret, int verify_ret
     TEST_ASSERT(mbedtls_pk_setup(&pk, mbedtls_pk_info_from_type(type)) == 0);
     TEST_ASSERT(pk_genkey(&pk, curve_or_keybits) == 0);
 
+#if defined(MBEDTLS_RSA_C)
+    if (type == MBEDTLS_PK_RSA) {
+        /* Just pick SHA1 here as hashing algorithm as we're more interested
+         * in checking the compatibility between */
```

Comment:
Sentence looks incomplete. Anyway, this doesn't look like it's just picking SHA1: `rsa_md_alg` is an input parameter and could hold any value.